src/components/graph/graph.helper.js

```diff
@@ -331,6 +331,7 @@ function buildNodeProps(node, config, nodeCallbacks = {}, highlightedNode, highl
             node.id === (highlightedLink && highlightedLink.target));
     const opacity = _getNodeOpacity(node, highlightedNode, highlightedLink, config);
     let fill = node.color || config.node.color;
+    let labelProperty = node[config.node.labelProperty] || node.id;
```

Comment:
Please rename `labelProperty` to label and move this assignment next to the new logic that you added